src/content/app/help/Help.scss

```diff
@@ -30,7 +30,7 @@
   grid-template-areas:
     'breadcrumbs breadcrumbs navigation'
     '.  article  article';
-  grid-template-rows: 40px auto;
+  grid-template-rows: 40px calc(100% - 65px);
```

Comment:
By `calc(100% - 65px)`, do you mean `1fr`?

Reply:
It doesn't work with `1fr` unfortunately